packages/elements-core/src/components/TryIt/Servers/Servers.tsx

```diff
@@ -0,0 +1,44 @@
+import { Flex, Input, Panel, Text } from '@stoplight/mosaic';
+import { useAtom } from 'jotai';
+import * as React from 'react';
+
+import { chosenServerAtom } from '../chosenServer';
+import { chosenServerVariablesAtom } from '../chosenServerVariables';
+
+interface TryItServersProps {}
+
+export const TryItServers: React.FC<TryItServersProps> = () => {
+  const [chosenServer, _] = useAtom(chosenServerAtom);
+  const [chosenServerVariables, setChosenServerVariables] = useAtom(chosenServerVariablesAtom);
+
+  if (!chosenServer || !chosenServer.variables || Object.entries(chosenServer.variables).length === 0) {
+    return null;
+  }
+
+  return (
+    <Panel defaultIsOpen>
+      <Panel.Titlebar>Server</Panel.Titlebar>
+      <Panel.Content className="ParameterGrid">
+        {Object.entries(chosenServer.variables).map(([key, value]) => (
+          <React.Fragment key={key}>
+            <div>{key}</div>
+            <Text mx={3}>:</Text>
+            <Flex flex={1}>
+              <Input
```

Comment:
If the spec provides an enum of valid values for a specific variable, we should expose a dropdown select populated with those enum values, rather than an Input.